combat-trainer.lic

```diff
@@ -147,7 +147,7 @@ class SetupProcess
     return if Time.now - @last_cycle_time < @cycle_armors_time
     return if game_state.loaded
     armor_types = @cycle_armors.map { |skill, _| skill }
-    next_armor_type = game_state.sort_by_rate_then_rank(armor_types).first
+    next_armor_type = armor_types.min_by { |skill| [DRSkill.getxp(skill), DRSkill.getrank(skill)] }
```

Comment:
⚡ Merge conflict, this is inadvertently reverting changes introduced in https://github.com/rpherbig/dr-scripts/pull/4686 that should be kept